pkgs/development/compilers/polyml/default.nix

```diff
@@ -18,6 +18,10 @@ stdenv.mkDerivation rec {
     "--with-gmp"
   ];
 
+  buildPhase = ''
+    make compiler
```

Comment:
Please use makeFlags.

Reply:
I don't think that's right, that would mean that the `compiler` target gets invoked during the install phase too.

Reply:
Why? Using ``makeFlags = [ "compiler" ];`` should do the same.

Reply:
`makeFlags = [ "compiler" ];` would do `make compiler` during `buildPhase` and `make compiler install` in the `installPhase`, which is wrong. However, I have found the `buildFlags` option, I will use it instead.